pages/[...slug].js

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs'
+import matter from 'gray-matter'
+import hydrate from 'next-mdx-remote/hydrate'
+import renderToString from 'next-mdx-remote/render-to-string'
+import dynamic from 'next/dynamic'
+import path from 'path'
+import remarkExternalLinks from 'remark-external-links'
+import remarkImages from 'remark-images'
+import remarkFootnote from 'remark-numbered-footnote-labels'
+import remarkUnwrapImages from 'remark-unwrap-images'
+import remarkWikiLink from 'remark-wiki-link'
+import remarkInlineLinks from 'remark-inline-links'
+import AnchorTag from '../components/AnchorTag'
+import CodeBlock from '../components/CodeBlock'
+import Image from '../components/Image'
+import { postFilePaths, POSTS_PATH } from '../utils/mdxUtils'
+
+// prettier-ignore
+const components = {
+    a: props => <AnchorTag {...props} />,
+    code: CodeBlock,
+    img: (props) => <div className="nextImageWrapper"><Image  {...props} /></div>,
+    Image: (props) => <div className="nextImageWrapper"><Image  {...props} /></div>,
+    Button: dynamic(() => import('../components/button')),
+}
+
+export default function PostPage({ source, frontMatter }) {
+  const content = hydrate(source, { components })
+  return content
+}
+
+export const getStaticProps = async ({ params }) => {
+  let postPath = 'index'
+
+  if (params && params.slug) {
+    postPath = Array.isArray(params.slug) ? params.slug.join('/') : params.slug
+  }
+
+  const postFilePath = path.join(POSTS_PATH, `${postPath}.mdx`)
+  const source = fs.readFileSync(postFilePath)
+
+  const { content, data } = matter(source)
+
+  const mdxSource = await renderToString(content, {
+    components,
+    mdxOptions: {
+      remarkPlugins: [
+        remarkImages,
+        remarkExternalLinks,
+        remarkFootnote,
+        remarkInlineLinks,
+        remarkUnwrapImages,
+        [
+          remarkWikiLink,
+          {
+            hrefTemplate: (permalink) => `${permalink}`,
```

Comment:
Looks like here is where remarkWikiLink is not returning the full path. Line 56 outputs `/note` instead of `/note/note.` A possible fix is to construct the correct href from `postFilePath.`